libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -43,7 +43,12 @@
 #define AP_ARMING_COMPASS_MAGFIELD_MAX  875     // 1.65 * 530 milligauss
 #define AP_ARMING_BOARD_VOLTAGE_MAX     5.8f
 #define AP_ARMING_ACCEL_ERROR_THRESHOLD 0.75f
-#define AP_ARMING_AHRS_GPS_ERROR_MAX    10      // accept up to 10m difference between AHRS and GPS
+#define AP_ARMING_AHRS_GPS_ERROR_MAX    10      // accept up to 10m difference between AHRS and GPS horizontally
+
+#define AP_ARMING_AHRS_GPS_ERROR_Z_MAX_CM   5000      // accept up to 50m difference between AHRS and GPS vertically
+#if !APM_BUILD_TYPE(APM_BUILD_APMrover2) // Rover doesn't get altitudes correct
+const uint16_t AP_ARMING_BARO_GPS_ERROR_Z_MAX_CM = 5000;      // accept up to 50m difference between BARO and GPS vertically
```

Comment:
I'd expand the range on this, as long flights with baro drift can reasonably hit this, without this being a panic condition. I'd suggest more like 100m to 250m.

Also it's a define, no need to guard against build type.

Reply:
It's now within the existing guard (I'd argue the previous guard was good as well...)

I've also increased the allowed error to 100m